examples/physics/basic/example.js

```diff
@@ -0,0 +1,207 @@
+let ground;
+let rightWall;
+let leftWall;
+
+let ball;
+
+let gravity;
+let prevHandPosition;
+
+let triggerPreviouslyHeld = false;
+
+
+function preload() {
+  createVRCanvas();
+}
+
+function setup() {
+  setVRBackgroundColor(0,0,0);
+  ground = new Boundary(0, -3, 0, 20, 1, 20, 1, -1, 1);
+  leftWall = new Boundary(-10, 0, 0, 1, 20, 20, -1, 1, 1);
+  rightWall = new Boundary(10, 0, 0, 1, 20, 20, -1, 1, 1);
+
+  gravity = createVector(0, -0.0002, 0);
+
+  ball = new Ball(1, 1, 0);
+}
+
+function draw() {
+  // orbitControl();
+
+  // pointLight(250, 250, 250, 0, 0, 0);
```

Comment:
Maybe add this back in and use `emissiveMaterial(this.color)` in Ball.render. This makes it easier to see the edge between boundaries.